packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -385,8 +390,21 @@ def imshow(
                 "The length of the x vector must match the length of the second "
                 + "dimension of the img matrix."
             )
+
+        if text_auto is True:
+            texttemplate = "%{z}"
+        elif text_auto is not False:
+            texttemplate = "%{z:" + text_auto + "}"
```

Comment:
does this need an `else` since we're definitely using `texttemplate` below?

Reply:
yes, thanks!